src/gui/include/gui/gui.h

```diff
@@ -584,6 +584,10 @@ class Gui
   void saveDisplayControls();
   void restoreDisplayControls();
 
+  // Used to add and remove focus nets from layout
+  void addFocusNet(odb::dbNet* net);
+  void removeFocusNet(odb::dbNet* net);
```

Comment:
I think it would be better if we had a clearFocusNet than an implied clear with removeFocusNet(nullptr).

Reply:
added